doc/aspice_swe3_4/swe_docu_guidelines.md

```diff
@@ -0,0 +1,52 @@
+# [ASPICE](http://www.automotivespice.com/) example [WIP]
+
+## Purpose
+
+This document defines the documentation guidelines for SWE 3 (Detailed Design) and SWE 4 (Unit Verification).
+
+
+## ToDo
+ - which doxygen tags needs to be set
+ - clarify alias for concurrent and error
+ - Doxygen plantuml example
+ - write documentation guideline
+
+##  Tags
+### Allowed Tags
+ - someTag
+
+### Custom Tags
+
+### Forbidden Tags
+- unit , use a type safe implementation if possible!
+- min , use a type safe implementation if possible!
```

Comment:
we should create a `Range` class. Combined with `NewType` this would be quite powerful.